### PR TITLE
D8 nid 821 b

### PR DESCRIPTION
**Pull request by DuttonMa**
Add processing for the new article teasers custom block that may be added by layout builder (on landing pages).
The old article teasers block is now obsolete so some code has been removed and some has been re-purposed.
Related PRs:
https://github.com/dof-dss/nicsdru_nidirect_theme/pull/148
https://github.com/dof-dss/nidirect-drupal/pull/417

### CODE REVIEW

File: nidirect_landing_pages/nidirect_landing_pages.module [hunk 1]
```diff
@@ -42,134 +44,29 @@ function nidirect_landing_pages_entity_update(EntityInterface $entity) {
...
-    }
-    elseif ($this_route == 'entity.node.edit_form') {
-      $node = \Drupal::routeMatch()->getParameter('node');
-      if (is_object($node) && ($node instanceof Node) && ($node->getType() == 'landing_page')) {
-        // We are editing a landing page, add js to handle special behaviour.
-        $attachments['#attached']['library'][] = 'nidirect_landing_pages/landing_listing';
+  $this_route = \Drupal::routeMatch()->getRouteName();
+  if ($this_route == 'layout_builder.overrides.node.view') {
+    $attachments['#attached']['library'][] = 'nidirect_landing_pages/lb_config_teasers';
+    $attachments['#attached']['library'][] = 'nidirect_landing_pages/lb_config_teasers_css';
```
Comment (omahm) [inline nidirect_landing_pages/nidirect_landing_pages.module:56]: Is there a reason for the 2 library entries?

**Review by omahm — COMMENTED** [review]
Impressive, quite a lot has gone into this. Just the 1 query around the separay library entries for Js and CSS

**Review by johangant — APPROVED** [review]
Lots of great work here 👍

File: nidirect_landing_pages/nidirect_landing_pages.module (hunk 1 above)
Comment (DuttonMa) [reply nidirect_landing_pages/nidirect_landing_pages.module]: Good point, have now condensed them into one

**Pull request merged**